commit 9144e39d252cd75dd2d6941154e014f7d46147ca
Author: John Johansen <john.johansen@canonical.com>
Date:   Fri Jun 14 01:04:22 2019 -0700

    Revert "utils/test-network.py: fix failing testcase"
    
    This reverts commit 378519d23f8b6e55b1c0741e8cd197863e0ff8a0.
    this commit was meant for the 2.13 branch not master
    
    Signed-off-by: John Johansen <john.johansen@canonical.com>

diff --git a/utils/test/test-network.py b/utils/test/test-network.py
index 6088327a..ee325abe 100644
--- a/utils/test/test-network.py
+++ b/utils/test/test-network.py
@@ -31,7 +31,7 @@ exp = namedtuple('exp', ['audit', 'allow_keyword', 'deny', 'comment',
 
 class NetworkKeywordsTest(AATest):
     def test_network_keyword_list(self):
-        rc, output = cmd(['make', '-s', '--no-print-directory', 'list_af_names'])
+        rc, output = cmd('../../common/list_af_names.sh')
         self.assertEqual(rc, 0)
 
         af_names = []
